draft-dns-zone-digest.xml

```diff
@@ -839,6 +840,7 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
     </section -->
 
     <section title="IANA Considerations" anchor="iana">
+      <!-- Whats requirement for adding new entries?  RFC or expert review? -->
```

Comment:
I think it is Specification Required -- originally I was saying Expert Review, but with the scheme, one needs to make sure that implementers know how to sort / canonicalize the input.